### PR TITLE
Jalena Penaligon

### PR DESCRIPTION
**Pull request by jalena-penaligon**

### CODE REVIEW

File: lib/vendor.rb [hunk 1]
```diff
@@ -4,7 +4,15 @@ class Vendor
 
   def initialize(name)
     @name = name
-    @inventory = # you decide what goes here...
+    @inventory = Hash.new(0)
```
Comment (ameseee) [inline lib/vendor.rb]: Nice job using this rather than `{}`!

File: test/market_test.rb [hunk 2]
```diff
@@ -5,4 +5,166 @@
...
+    assert_equal "South Pearl Street Farmers Market", market.name
+  end
+
+  def test_vendors_starts_empty
+    market = Market.new("South Pearl Street Farmers Market")
+
+    assert_equal [], market.vendors
+  end
+
+  def test_you_can_add_vendors
```
Comment (ameseee) [inline test/market_test.rb]: May want to consider adding an in edge case test that a vendor can only be added in once? This isn't something we've really talked about in class/expect, just trying to push you forward as you have this all down😊

**Comment by ameseee** [comment]
Hi @jalena-penaligon - this looks awesome💪 Just a couple comments for your review.
